src/test/java/Pages/BasePage.java

```diff
@@ -0,0 +1,27 @@
+package Pages;
```

Comment:
A package in Java is used to group related classes. 

For example package "pages" - Includes the Page Objects (or Page Classes)